src/transport.ts

```diff
@@ -55,7 +55,7 @@ export class Transport {
     this.apiKMSKey = apiKMSKey;
   }
 
-  setEnvVars(lambdas: lambda.Function[]) {
+  applyEnvVars(lambdas: lambda.Function[]) {
```

Comment:
Why was this renamed?

Reply:
This was also because of a JSII error:
```
[2021-03-22T14:59:21.350] [ERROR] jsii/compiler - Type model errors prevented the JSII assembly from being created
error JSII5001: Methods and properties cannot have names like "setXxx": those conflict with Java property setters. Rename "Transport.setEnvVars"
```
I guess functions that start with "set" confict with Java setter functions so its just a quirk so that JSII can do its compiling correctly.